site/content/en/docs/tasks/run_with_kubeflow/run_pytorchjobs.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Run a PyTorchJob"
+date: 2023-08-09
+weight: 6
+description: >
+  Run a Kueue scheduled PyTorchJob
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Training Operator](https://github.com/kubeflow/training-operator) MPIJobs.
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+Check [administer cluster quotas](/docs/tasks/administer_cluster_quotas) for details on the initial cluster setup.
+
+Check [the Training Operator installation guide](https://github.com/kubeflow/training-operator#installation).
+
+Set PyTorchJobs as an allowed workload in Kueue Configuration.
+
+```yaml
+integrations:
+  frameworks:
+  - "kubeflow.org/pytorchjob" 
+```
+
+You can [modify kueue configurations from installed releases](/docs/installation#install-a-custom-configured-released-version) to include PyTorchJobs as an allowed workload.
+
+## PyTorchJob definition
+
+### a. Queue selection
+
+The target [local queue](/docs/concepts/local_queue) should be specified in the `metadata.labels` section of the PyTorchJob configuration.
+
+```yaml
+metadata:
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+### b. Optionally set Suspend field in PyTorchJobs
+
+```yaml
+spec:
+  runPolicy:
+    suspend: true
+```
+
+By default, Kueue will set `suspend` to true via webhook and unsuspend it when the PyTorchJob is admitted.
+
+## Sample PyTorchJob
+
+This example is based on https://github.com/kubeflow/training-operator/blob/855e0960668b34992ba4e1fd5914a08a3362cfb1/examples/pytorch/simple.yaml.
+
+Note that the minimum requirement training-operator version is v1.7.0.
+
+```yaml
```

Comment:
use `{{% include "sample-pytorchjob.yaml" "yaml" %}}`

Reply:
It's an awsome suggestion!

Reply:
Done.

Reply:
Uhm, build failed due to this change 🤔

Reply:
```
error calling readFile: stat /opt/build/repo/site/docs/static/examples/sample-pytorchjob.yaml: no such file or directory
```
is there a typo? I don't see it.

Reply:
There aren't any typos.

There is an example folder in the `/site/static/examples`.
However, build log say `/site/docs/static/examples` 🤔

Reply:
I see. The file `site/layouts/shortcodes/include.html` uses `../../static/examples`.

We need to find a way to use absolute paths.

Reply:
I see. Let me check.

Reply:
Done.